docs/openapi.yaml

```diff
@@ -525,6 +525,47 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Problem'
+    post:
+      summary: Create Publication from Import candidate
+      parameters:
+        - in: path
+          name: importCandidateIdentifier
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: UUID identifier of the import candidate to publish.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ImportCandidate'
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${CreatePublicationFromImportCandidateFunction.Arn}:live/invocations
+        responses: { }
+        httpMethod: POST
+        type: AWS_PROXY
+      responses:
+        '200':
```

Comment:
IMO you are missing some error codes:

If you try to POST a publication that is not exist as an import-candidates. ex. the post has a scopus-identifier, but the publication identifier does not match in the database. Bad Request.

The publication is missing a scopus identifier. Bad Request.

You are trying to import something that already has been imported: Error code or 200 code?

Also the appropriate status code for for  a POST method is 201